src/apps/ocioconvert/main.cpp

```diff
@@ -13,6 +13,16 @@ namespace OCIO = OCIO_NAMESPACE;
 
 #include <OpenImageIO/imageio.h>
 #include <OpenImageIO/typedesc.h>
+
+// Take the half.h the same way OpenImageIO takes it i.e. do not use the Imath/OpenEXR one from
+// OpenColorIO to avoid version clashes between OpenColorIO & OpenImageIO libraries. For example,
+// OpenColorIO uses Imath 3.1.x but OpenImageIO (from the system) is using Imath 3.0.x which
+// breaks the OpenColorIO compilation.
+#if (OIIO_VERSION >= 20200)
```

Comment:
@lgritz, Based on my limited experiments, the include file `<OpenImageIO/Imath.h>` only exists starting with oiio version 2.2. Let me know if I need to change/improve the check.

Reply:
That's right. I believe I never backported support for Imath 3.x to OIIO 2.1.

Reply:
I never know the right answer to this problem. In practice, I'm very fastidious about making sure that the "current" release family (2.2 until yesterday, now 2.3!) is constantly kept able to build with the latest compilers and dependencies. But once a few months have elapsed since a big release, I tend to only backport things like to the "obsolete" branches if somebody requests it. Since VFX Platform 2021 is still on OpenEXR 2.4 (!), VP2022 will be the first to dictate OpenEXR/Imath >= 3.0, the popular demand for that backport to 2.1 has not materialized.